nidx/nidx_protos/noderesources.proto

```diff
@@ -164,11 +164,13 @@ message Resource {
     map<string, IndexParagraphs> paragraphs = 6; // Paragraphs by field
 
     repeated string paragraphs_to_delete = 7;
-    // DEPRECATED: replaced by vector_prefixes_to_delete in order to support
-    // updates on a subset of the available vectorsets
-    repeated string sentences_to_delete = 8 [deprecated = true];
+
+    // Delete in all vectorsets (e.g: resource deletion)
+    repeated string vectors_to_delete_in_all_vectorsets = 8;
```

Comment:
This is actually still in use when deleting a resource, so having it as deprecated makes little sense.